maths/juggler_sequence.py

```diff
@@ -0,0 +1,57 @@
+"""
+== Juggler Sequence ==
+Juggler sequence start with any positive integer n. The next term is
+obtained as follows:
+    If n term is even, the next term is floor value of square root of n .
+    If n is odd, the next term is floor value of 3 time the square root of n.
+
+https://en.wikipedia.org/wiki/Juggler_sequence
+"""
+
+# Author : Akshay Dubey (https://github.com/itsAkshayDubey)
+import math
+
+
+def juggler_sequence(number: int) -> list[int]:
+    """
+    >>> juggler_sequence(0)
+    Traceback (most recent call last):
+        ...
+    ValueError: Input value of [number=0] must be a positive integer
+    >>> juggler_sequence(1)
+    [1]
+    >>> juggler_sequence(2)
+    [2, 1]
+    >>> juggler_sequence(3)
+    [3, 5, 11, 36, 6, 2, 1]
+    >>> juggler_sequence(5)
+    [5, 11, 36, 6, 2, 1]
+    >>> juggler_sequence(6.0)
+    Traceback (most recent call last):
+        ...
+    TypeError: Input value of [number=6.0] must be an integer
+    >>> juggler_sequence(-1)
+    Traceback (most recent call last):
+        ...
+    ValueError: Input value of [number=-1] must be a positive integer
+    """
+    if not isinstance(number, int):
+        raise TypeError(f"Input value of [number={number}] must be an integer")
+    if number < 1:
+        raise ValueError(f"Input value of [number={number}] must be a positive integer")
+    sequence = [number]
+    while number != 1:
+        if number % 2 == 0:
+            number = int(math.floor(math.sqrt(number)))
```

Comment:
```suggestion
            number = math.floor(math.sqrt(number))
```
`math.floor` returns `int`

Reply:
Fixed in [this](https://github.com/TheAlgorithms/Python/pull/7985/commits/dc5db724bd939e08a44ae13fa9030be0734b8582) commit. Kindly have a look.